packages/trader/src/Modules/SmartChart/Components/top-widgets.jsx

```diff
@@ -59,6 +60,7 @@ const TopWidgets = ({
                     width: `calc(100% - ${y_axis_width ? y_axis_width + 5 : 0}px)`,
                 }}
             >
+                {is_mobile && <BuyToastNotification portal_id='popup_root' />}
```

Comment:
```suggestion
                {is_mobile && <BuyToastNotification />}
```
@hamza-deriv The default value for `portal_id` already is `popup_root`, Maybe we can even remove the prop from the component if it's not needed 🤔

Reply:
yes can be removed